Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -215,53 +246,53 @@ internal final class ReflectionServiceProvider: Reflection_ServerReflectionAsync
     }
     return Reflection_ServerReflectionResponse(
       request: request,
-      listServicesResponse: listServicesResponse
+      messageResponse: Reflection_ServerReflectionResponse.OneOf_MessageResponse
+        .listServicesResponse(
+          listServicesResponse
+        )
```

Comment:
Let's use type inference here:
```suggestion
      messageResponse: .listServicesResponse(listServicesResponse)
```